api/client/nic.cc

```diff
@@ -437,6 +441,8 @@ static void SubNicAddHelp(void) {
             << endl <<
         "    --enable-antispoof  enable antispoof protection (default: off)"
             << endl <<
+        "    -t true/false or --packet-trace true/false  enable traffic in" <<
```

Comment:
replace "enable traffic in pcap file" by "choose whatever we trace a nic (default: use default server behaviour)"